spec/system/placements/schools/placements/view_placements_list_spec.rb

```diff
@@ -53,6 +53,29 @@
       given_i_sign_in_as_anne
       then_i_see_term_name("Autumn term")
     end
+
+    context "when using the academic year navigation" do
+      let(:current_academic_year) { Placements::AcademicYear.current }
+      let(:next_academic_year) { current_academic_year.next }
+      # Subjects were added to make sure tha placement titles are always unique.
+      let(:current_subject) { create(:subject, name: "Current academic year studies") }
+      let(:next_subject) { create(:subject, name: "Next academic year studies") }
+      let!(:current_academic_year_placement) { create(:placement, school:, academic_year: current_academic_year, subject: current_subject) }
+      let!(:next_academic_year_placement) { create(:placement, school:, academic_year: next_academic_year, subject: next_subject) }
+
+      scenario "when I view placements for the current academic year" do
+        given_i_sign_in_as_anne
+        then_i_see_placement(current_academic_year_placement)
+        and_i_do_not_see_placement(next_academic_year_placement)
+      end
+
+      scenario "when I view placements for the next academic year" do
+        given_i_sign_in_as_anne
+        and_i_click_on("Next year (#{next_academic_year.name})")
+        then_i_see_placement(next_academic_year_placement)
+        and_i_do_not_see_placement(current_academic_year_placement)
+      end
```

Comment:
If I'm interpreting the system specs correctly, I think we test the paths:

- `/placements`
- `/placements?year=next`

But we never actually test:

- `/placements?year=current`

I was trying to work out why that doesn't result in a coverage failure. But it turns out it's because:

```ruby
def academic_year_scope
  params[:year] == "next" ? current_academic_year.next : current_academic_year
end
```

This code :point_up: only has two branches. Either `params[:year] == "next"` or it doesn't.

So we're implicitly covering it already. Although I do think it might be nice so explicitly click the "This year" tab just to prove that it works.

Reply:
What do you think about something like this?

```ruby
scenario "I can switch between academic years" do
  given_i_sign_in_as_anne

  then_i_see_placement(current_academic_year_placement)
  and_i_do_not_see_placement(next_academic_year_placement)

  when_i_click_on("Next year (#{next_academic_year.name})")
  then_i_see_placement(next_academic_year_placement)
  and_i_do_not_see_placement(current_academic_year_placement)

  when_i_click_on("This year (#{current_academic_year.name})")
  then_i_see_placement(current_academic_year_placement)
  and_i_do_not_see_placement(next_academic_year_placement)
end
```